nifi-nar-bundles/nifi-media-bundle/nifi-media-processors/src/test/java/org/apache/nifi/processors/image/ExtractImageMetadataTest.java

```diff
@@ -37,7 +37,7 @@
     private static String BMP_HEADER = "BMP Header.";
     private static String JPEG_HEADER = "JPEG.";
     private static String GIF_HEADER = "GIF Header.";
-    private static String PNG_HEADER = "PNG.";
+    private static String PNG_HEADER = "PNG-";
```

Comment:
Ok.  Even that punts the problem because it just means we'll stick with an older dependency.  My vote here would be we keep it as-is.  Update the migration guide to call this issue out when going to 0.7 (and 1.0).  And we update the processor documentation to highlight that the attributes extracted by this processor are not stable and subject to change across upgrades.  It isn't great but it is honest.